File: common/src/main/java/com/jmsoftware/maf/common/bean/TrackableBean.java
package com.jmsoftware.maf.common.bean;

import lombok.Data;
import org.springframework.lang.Nullable;

/**
 * <h1>TrackableBean</h1>
 * <p>
 * Change description here.
 *
 * @author Johnny Miller (鍾俊), email: johnnysviva@outlook.com, 3/12/22 1:17 AM
 **/
@Data
public class TrackableBean {
    /**
     * The Track. Pattern: [Zipkin trace id]#[Span id]
     */
    @Nullable
    private String track;
}

File: common/src/main/kotlin/com/jmsoftware/maf/common/bean/TrackableBean.kt
package com.jmsoftware.maf.common.bean

/**
 * # TrackableBean
 *
 * Change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/12/22 9:50 PM
 */
open class TrackableBean {
    /**
     * The Track. Pattern: [Zipkin trace id]#[Span id]
     */
    var track: String? = null
}
